app/src/main/java/com/codesoom/assignment/handlers/DemoHttpHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.codesoom.assignment.handlers;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+
+public class DemoHttpHandler implements HttpHandler {
```

Comment:
데모 용도의 아주 작은 클래스를 잘 만드셨어요. 여기에 JavaDoc을 작성하는 것도 한번 해볼까요? 이 클래스 위에 이렇게 적어주면 됩니다. 내용은 이 클래스를 잘 설명하는 문장으로 고민해서 작성해 보세요.

```suggestion
/**
 * 내용.
 */
public class DemoHttpHandler implements HttpHandler {
```